Formula/a/anchor.rb

```diff
@@ -0,0 +1,19 @@
+class Anchor < Formula
+  desc "Solana Program Framework"
+  homepage "https://anchor-lang.com"
+  url "https://github.com/solana-foundation/anchor/archive/refs/tags/v0.31.1.tar.gz"
+  sha256 "0c9b1e3e1f14e78cb00271171b1cfac177c7c887814b022196bcbf7e2389e089"
+  license "Apache-2.0"
+
+  depends_on "rust" => :build
+  depends_on "node"
+  depends_on "yarn"
```

Comment:
are these runtime dependencies?

also saw the node used in the upstream dockerfile is with [node 20](https://github.com/solana-foundation/anchor/blob/master/docker/build/Dockerfile#L13)?